.github/ISSUE_TEMPLATE/---bug-report.md

```diff
@@ -0,0 +1,28 @@
+---
+name: "\U0001F41B Bug report"
+about: Create a report to help us improve
+title: ''
+labels: bug report, needs-triage
+assignees: ''
+
+---
+
+Confirm by changing [ ] to [x] below to ensure that it's a bug:
+- [ ] I've searched for [previous similar issues](https://github.com/aws/aws-iot-device-sdk-python-v2/issues) and didn't find any solution
+  
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**SDK version number**
+
+**To Reproduce (observed behavior)**
+Steps to reproduce the behavior (please share code)
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
```

Comment:
Maybe rephrase to something like "Logs / output / screenshots". I'd hate someone think we'd rather see a screenshot of their logs, than the ascii value of their logs.

Reply:
Can we suggest that they turn on logging?
`awscrt.io.init_logging(awscrt.io.LogLevel.Warn, 'stderr')`

Reply:
Yea, makes sense more sense that way